dll/win32/kernel32/winnls/locale_rc.rc

```diff
@@ -15,377 +15,248 @@
  *
  * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
-#include <windef.h>
-//#include "winbase.h"
-#include <winnls.h>
+#include "windef.h"
+#include "winbase.h"
+#include "winnls.h"
```

Comment:
Why in quotes? These files are from the "main" headers (PSDK) -> should be better between chevrons?

Reply:
Same answer as above